### PR TITLE
[WIP] Set up Collector Marketing Page

### PR DESCRIPTION
**Pull request by yuki24**
Signed-off-by: Yuki Nishijima <yuki@artsymail.com>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please assign someone to your PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: desktop/apps/buy_art/routes.coffee [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+resizer = require '../../components/resizer'
+markdown = require '../../components/util/markdown'
+Items = require '../../collections/items'
+
+landing = new JSONPage name: 'buy-art/landing'
+
+@landing = (req, res, next) ->
+  landing.get()
+    .then (data) ->
+      res.render 'landing', extend {},
```
Comment (alloy) [inline desktop/apps/buy_art/routes.coffee]: What is this `extend {}` for?

**Comment by yuki24** [comment]
closing in favor of #1004.

File: desktop/apps/buy_art/routes.coffee (hunk 1 above)
Comment (yuki24) [inline desktop/apps/buy_art/routes.coffee]: I'm not actually sure, and couldn't find any doc either. I'm guessing it's something like [lodash's extend](https://lodash.com/docs/#assignIn), though.

@xtina-starr Do you know anything about `extend`?

**Pull request closed without merging**